src/components/calcite-alert/calcite-alert.tsx

```diff
@@ -120,13 +120,18 @@ export class CalciteAlert {
         ref={(el) => (this.closeButton = el)}
         type="button"
       >
-        <calcite-icon icon="x" scale="m" />
+        <calcite-icon icon="x" scale={this.scale === "s" ? "s" : "m"} />
       </button>
     );
+    const queueText = `+${this.queueLength > 2 ? this.queueLength - 1 : 1}`;
```

Comment:
Should this text be localized in some kind of number formatting?

Reply:
It wasn't formatted to begin with, so isn't that out of scope for this pr? Otherwise, can you clarify what you're looking for?

Reply:
Yeah its out of scope but probably an issue should be created to localize this number. Otherwise, I don't think it will look right in other languages.

Reply:
I'll create a follow up issue.

Reply:
got it, thanks @driskull